src/redux/actions/MenteeRegistration.redux/reducer.js

```diff
@@ -0,0 +1 @@
+initialState = {};
```

Comment:
Ignoring the fact that this does nothing and is not used anywhere, it needs to be prefixed with `var`, `let`, or preferrably `const`.

Reply:
Should probably rename this file from `reducer.js` to `MenteeRegistration.reducer.js` to conform to the pattern already established in other `.redux` folders.